src/main/java/flaxbeard/steamcraft/tile/TileEntitySteamCharger.java

```diff
@@ -308,8 +308,10 @@ private float getChargingPercent(ItemStack stack) {
             int maxFill = stack.stackTagCompound.getInteger("maxFill");
             int steamFill = stack.stackTagCompound.getInteger("steamFill");
             return ((float) steamFill / (float) maxFill);
+        } else if (stack != null) {
```

Comment:
I think this is kind of confusing. It would be make more sense, to me, to change this whole method to be more like:

```java
if (stack == null) {
    return 0F;
}
if (stack.getItem() instanceof ItemExosuitArmor) {
    // ... stuff
    return ((float) steamFill / (float) maxFill);
}
return 1F - ((float) stack.getItemDamage() / (float) stack.getMaxDamage());
```
```